spec/models/spree/line_item_spec.rb

```diff
@@ -101,7 +101,7 @@ module Spree
       end
 
       it "caps at zero when stock is negative" do
-        v.update_attributes! on_hand: -2
+        v.__send__(:stock_item).update_column(:count_on_hand, -2)
```

Comment:
Why is `__send__` needed here? Doesn't `variant.stock_item` work?

Reply:
@Matt-Yorkley Yes, it turns out `Spree::Variant#stock_item` is a private method.